edelphi/src/main/resources/fi/metatavu/edelphi/liquibase/query-page-setting-keys.sql

```diff
@@ -0,0 +1,61 @@
+INSERT INTO  
+  QueryPageSettingKey (id, name)
+VALUES   
+  (1, 'text.content'),
+  (2, 'text.commentable'),
+  (3, 'text.viewDiscussions'),
+  (4, 'thesis.commentable'),
+  (5, 'thesis.viewDiscussions'),
+  (6, 'thesis.showLiveReport'),
+  (7, 'thesis.text'),
+  (8, 'thesis.description'),
+  (9, 'multiple2dscales.labelx'),
+  (10, 'multiple2dscales.labely'),
+  (11, 'multiple2dscales.options'),
+  (12, 'multiple2dscales.theses'),
+  (13, 'time_serie.maxY'),
+  (14, 'time_serie.maxX'),
+  (15, 'time_serie.minY'),
+  (16, 'time_serie.minX'),
+  (17, 'time_serie.stepX'),
+  (18, 'time_serie.stepY'),
+  (19, 'time_serie.userStepX'),
+  (20, 'time_serie.yTickDecimals'),
+  (21, 'time_serie.xAxisTitle'),
+  (22, 'time_serie.yAxisTitle'),
+  (23, 'time_serie.predefinedSetLabel'),
+  (24, 'time_serie.userSetLabel'),
+  (25, 'time_serie.predefinedValues'),
+  (26, 'scale2d.label.x'),
+  (27, 'scale2d.label.y'),
+  (28, 'scale2d.type'),
+  (29, 'scale2d.options.x'),
+  (30, 'scale2d.options.y'),
+  (31, 'grouping.groups'),
+  (32, 'grouping.items'),
+  (33, 'scale1d.label'),     
+  (34, 'scale1d.type'),      
+  (35, 'scale1d.options'),   
+  (36, 'visible'),          
+  (37, 'expertise.showLiveReport'),
+  (38, 'timeline.value1Label'),
+  (39, 'timeline.value2Label'),
+  (40, 'timeline.min'),     
+  (41, 'timeline.max'),     
+  (42, 'timeline.step'),    
+  (43, 'multiselect.options'),
+  (44, 'orderingField.items'),
+  (45, 'form.fields'),      
+  (46, 'timeline.type'),    
+  (47, 'collage2d.includedPages'),
+  (48, 'collage2d.pageSettings'),
+  (49, 'collage2d.replySource'),
+  (50, 'collage2d.labelVisibility'),
+  (51, 'collage2d.replyCountsVisible'),
+  (52, 'collage2d.dotOffset'),
+  (53, 'collage2d.dotMultiplier'),
+  (54, 'expertise.description')
+ON DUPLICATE KEY UPDATE name = VALUES(name);
+
+DELETE FROM hibernate_sequences WHERE sequence_name in ('QueryPageSettingKey');
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The reserved word "in" should be written in upper case. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AUpperCaseReservedWordsCheck)